jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -510,10 +512,14 @@ ecma_deref_ecma_string (ecma_string_t *string_p) /**< ecma-string */
     case ECMA_STRING_CONTAINER_MAGIC_STRING_EX:
     {
       /* only the string descriptor itself should be freed */
+      break;
+    }
+    case ECMA_STRING_CONTAINER__COUNT:
```

Comment:
It's unnecessary to have this case explicitly refer to `ECMA_STRING_CONTAINER__COUNT`. A `default:` branch would suite here perfectly. (Actually, a default unreachable assert was missing from here, unrelated to the current change. I don't know why this was forgotten originally.)